server/public_routes.go

```diff
@@ -54,6 +54,10 @@ func PublicRoutes(app *app.App) []*route.HandledRoute {
 		route.Delete("/totp").
 			SecuredWith(originSecurity).
 			Handle(handlers.DeleteTOTP(app)),
+
+		route.Get("/oauth/info").
```

Comment:
Maybe a better route name would be `/oauth/accounts` or even just `/oauth`

Reply:
I like the idea of info here, make easier to extend datapoints.

Reply:
`/oauth/info` sounds like an endpoint that will tell me about the server's oauth configuration.

If we want to introduce "tell me about my own account" endpoints, then here are a couple ideas:

* `/account`
* `/me`

Both might return the same payload as `/account/:id`.

Reply:
Thanks @AlexCuse and @cainlevy for the heads up, I have changed the implementation to use `/oauth/accounts`